README.md

```diff
@@ -92,6 +92,15 @@ Start mailcatcher with `$ mailcatcher` and navigate to
 Note that [mail_safe](https://github.com/myronmarston/mail_safe) maybe also be running in which
 case any development email will seem to be sent to your global git config email address.
 
+#### Setting an known state for our environments
+
+There are times when we require our environments to be cleared of
+projects and put into a known state. For these occassions we have the
+following task: `bundle exec projects:delete_all`.
+
+Please note that this will drop all projects, along with any associated
+data. It will however leave users and their areas intact.
```

Comment:
@baphled If you could expand on why here, that'd be great :)

Reply:
Updated, though it triggered Hakiri to complain about some vulnerabilities.

Add https://github.com/DEFRA/pafs-user/pull/147 to resolve those.